README.md

```diff
@@ -154,10 +154,39 @@ sdk "iphoneos9.0"
 
 clean true
 
+export_options ({
+  method: "ad-hoc",
```

Comment:
I see we already have an `export options` section, so it's okay to just remove this from here